pkg/controller/bookkeepercluster/bookie.go

```diff
@@ -328,23 +315,11 @@ func createVolumeMount(ledgerDirs []string, journalDirs []string, indexDirs []st
 		}
 		volumeMounts = append(volumeMounts, v)
 	}
-	if len(indexDirs) > 1 {
```

Comment:
why are we removing multiple index support?

Reply:
We never had the support. I am not sure why it was added here in the first place
https://github.com/pravega/pravega/blob/master/docker/bookkeeper/entrypoint.sh#L54

Reply:
I think that the question from @anishakj makes sense: the change in index seems completely unrelated and, if the Operator does not support multiple index directories, the natural change would be to just add them, right? In any case, if the change in the index volume is unrelated, please send it out as a separate PR. It is generally not a good idea to mix unrelated things in the same PR.